src/SignupForm.js

```diff
@@ -4,6 +4,17 @@ import './SignupForm.css';
 
 const getAjaxUrl = url => url.replace('/post?', '/post-json?')
 
+function formatLinks(msg) {
+  let dom = document.createRange().createContextualFragment(msg);
+  var links = dom.querySelectorAll("a");
+  links.forEach(function(link) {
+    link.setAttribute('target', '_blank');
+  });
+  let div=document.createElement("div");
+  div.appendChild(dom);
+  return div.innerHTML;
```

Comment:
why not just return `dom.innerHTML`?

Reply:
I initially tried this, but `dom.innerHTML` returns blank.